pkg/internalapis/cnsvolumeoperationrequest/v1alpha1/cnsvolumeoperationrequest_types.go

```diff
@@ -60,7 +60,7 @@ type QuotaDetails struct {
 	// +optional
 	Reserved *resource.Quantity `json:"reserved,omitempty"`
 	// StoragePolicyId is the ID associated with the storage policy.
```

Comment:
Can you update "StoragePolicyId" in the comments accordingly?

Reply:
Fixed